nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/AdminMain.java

```diff
@@ -215,10 +215,9 @@ public static void main(String[] args) {
     }
 
     protected int doMain(String[] args) {
-        int minor = JDK.getMinor();
-        int major = JDK.getMajor();
-        if (major < 11) {
-            System.err.println(strings.get("OldJdk", major, minor));
+        Version version = Runtime.version();
+        if (version.feature() < 11) {
```

Comment:
Just wondering - how this can be entered and "OldJdk" used, given
```
com/sun/enterprise/admin/cli/AdminMain.class: compiled Java class data, version 55.0 (Java SE 11)
```
and with 8, JVM of course refuses to start:
```
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/glassfish/admin/cli/AsadminMain has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
```

Reply:
It was in cases when ie in jdk 1.6u22 or so oracle added some features and gf needed them. I am not sure how dead this code is, but yeah, we can remove it and add it just when it would be required again.